superset/config.py

```diff
@@ -162,7 +161,9 @@ def _try_json_readsha(filepath):
 # Uncomment to specify where clicking the logo would take the user
 # e.g. setting it to '/welcome' would take the user to '/superset/welcome'
 LOGO_TARGET_PATH = None
-
+# Enables SWAGGER UI for superset openapi spec
+# ex: http://localhost:8080/swaggerview/v1
+FAB_API_SWAGGER_UI = True
 # Druid query timezone
 # tz.tzutc() : Using utc timezone
 # tz.tzlocal() : Using local timezone
```

Comment:
nit: I kinda like having a blank line highlighting separate sections. Would prefer to leave the old blank line after `LOGO_TARGET_PATH` and also adding a new one after `FAB_API_SWAGGER_UI`

Reply:
Makes sense